src/apps/rooms/models/domain/dabang.py

```diff
@@ -1,19 +1,14 @@
 from datetime import datetime
-from enum import IntEnum
 from typing import Any, List, Optional
 
 from pydantic import Field
 from pydantic.dataclasses import dataclass
 
 from ...exceptions import NoneTypeError
-from ..entity import BuildingType
+from ..entity import BuildingType, SellingType
 from .landlords import RoomItem
 
-
-class SellingType(IntEnum):
-    MonthlyRent = 0  # 월세
-    Jeonse = 1  # 전세
-    Selling = 2  # 매매
+image_url = "http://d1774jszgerdmk.cloudfront.net/512/{image_key}"
```

Comment:
ask;
다방의 image url인가요?

```suggestion
image_url = "http://d1774jszgerdmk.cloudfront.net/512/{image_key}"
```

Reply:
맞습니다 !